README.md

```diff
@@ -0,0 +1,13 @@
+# Gonum diff  [![Build Status](https://travis-ci.org/gonum/diff.svg)](https://travis-ci.org/gonum/diff)  [![Coverage Status](https://img.shields.io/coveralls/gonum/diff.svg)](https://coveralls.io/r/gonum/diff)
+
+Quad provides numerical integration of single variable functions by quadrature for the Go language.
```

Comment:
Quadarture is a synonym for numeric integration, but many users don’t know that. I like your wording. 

> On Nov 4, 2015, at 6:10 PM, Vladimír Chalupecký <notifications@github.com> wrote:
> 
> In README.md <https://github.com/gonum/quad/pull/1#discussion_r43965319>:
> 
> > @@ -0,0 +1,13 @@
> > +# Gonum diff  [![Build Status](https://travis-ci.org/gonum/diff.svg)](https://travis-ci.org/gonum/diff)  [![Coverage Status](https://img.shields.io/coveralls/gonum/diff.svg)](https://coveralls.io/r/gonum/diff)
> > +
> > +Quad provides numerical integration of single variable functions by quadrature for the Go language.
> Isn't quadrature a synonym for numerical integration? What about Quad provides numerical evaluation of univariate definite integrals for the Go language or ... of definite integrals of single-variable functions?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gonum/quad/pull/1/files#r43965319>.
>